### PR TITLE
Reset tic tac toe

### PR DESCRIPTION
**Pull request by saeidalidadi**
I added a reset button to example game in order to  warm up myself on using the framework,
You can merge it If you think this feature would be useful.

### CODE REVIEW

File: examples/tic-tac-toe/app.test.js [hunk 1]
```diff
@@ -15,6 +15,13 @@ Enzyme.configure({ adapter: new Adapter() });
 
 const Grid = (n) => Array(n).fill(null);
 
+const makeMove = (board, moves) => {
```
Comment (nicolodavis) [inline examples/tic-tac-toe/app.test.js]: Can you also use this down in line 70?

File: examples/tic-tac-toe/tic-tac-toe.js [hunk 2]
```diff
@@ -51,6 +53,11 @@ export const TicTacToe = Game({
       }
 
       return { ...G, cells };
+    },
+    resetGame(G) {
+      const cells = noFill(9);
+
+      return { ...G, cells }
```
Comment (nicolodavis) [inline examples/tic-tac-toe/tic-tac-toe.js]: Need to also reset `log` here.

File: examples/tic-tac-toe/app.test.js (hunk 1 above)
Comment (saeidalidadi) [reply examples/tic-tac-toe/app.test.js]: I can use also in line 37

File: examples/tic-tac-toe/tic-tac-toe.js (hunk 2 above)
Comment (saeidalidadi) [reply examples/tic-tac-toe/tic-tac-toe.js]: Ok

Comment (saeidalidadi) [reply examples/tic-tac-toe/tic-tac-toe.js]: the `log` list has not any reference inside `moves`

**Comment by nicolodavis** [comment]
This doesn't work correctly once the game has been won.

Also, it doesn't reset the turn (which you would expect to go back to 0). A better way to implement this would be to save the initial state (both G and ctx), and then use the `restore()` action available in the reducer.

**Comment by nicolodavis** [comment]
Going to close this for now given that it's old and out-of-sync. Feel free to re-open if you resume work on this.

**Pull request closed without merging**